zstacklib/zstacklib/utils/linux.py

```
@@ -1765,16 +1793,66 @@ def move_dev_route(src_dev, dest_dev):
+    parts = route.split()
+    if not parts:
+        return route
+    if 'dev' in parts:
+        index = parts.index('dev')
+        if index + 1 < len(parts):
+            parts[index + 1] = dev
+        return ' '.join(parts)
+    if 'via' in parts:
+        index = parts.index('via')
+        if index + 1 < len(parts):
+            return ' '.join(parts[:index + 2] + ['dev', dev] + parts[index + 2:])
```

Ruff (0.15.18)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)